src/web/lib/components/AppFooter/index.js

```diff
@@ -1,41 +1,77 @@
 import React from 'react';
 import ReactSVG from 'react-svg';
+
+import Metrics from '../../../../lib/metrics';
+
 import iconMoz from './moz-logo.svg';
 import iconGH from './github-logo.svg';
 import iconTwitter from './twitter-logo.svg';
 
 import './index.scss';
 
-export const AppFooter = () =>
+export const AppFooter = () => (
   <footer className="app-footer">
     <div className="app-footer__legal">
-      <a className="app-footer__legal-link" href="https://www.mozilla.org">
-        <ReactSVG path={ iconMoz } className="app-footer__legal-logo"/>
+      <a
+        className="app-footer__legal-link"
+        onClick={() => Metrics.linkClick('mozilla-logo')}
+        href="https://www.mozilla.org"
+      >
```

Comment:
Do we need `<a href="..." rel="noreferrer noopener">...</a>`, or is that only for `target="_blank"` links?
Or I guess, should we be adding `target="_blank"` to these links? (We have them in src/web/lib/components/AppHeader/index.js below)